src/content/1.0.0/guides/analytical_queries/full-text-search.md

```diff
@@ -93,7 +93,7 @@ Now, we can issue a full text query against any predicate or collection by using
 {
     "select": "?person",
     "where": [
-        ["?person", "fullText:person/handle", "jdoe"]
+        ["?person", "fullText:person/handle", "doe"]
```

Comment:
This is a semantic change as well, but I think it reflects the text above the example.

Reply:
this is the person/handle in the data set and is used multiple times. it should stay as jdoe.

Reply:
There's something weird going on here, I tried actually running the transactions and queries specified here and I'm not getting the same results. I've filed FC-1196 to track that.